### PR TITLE
Add integrations tab for easy access

### PR DESCRIPTION
**Pull request by SmritiSatyanV**
Fixed minor errors in docs
Added 'integrations' tab
Signed-off-by: SmritiSatyanV <smriti@union.ai>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -270,9 +270,9 @@ usually done by starting with:
 - [Github Issue](https://github.com/flyteorg/flyte/issues) - We maintain issues for all repos in the main flyte repo.
 - Writing down your proposal using a [Documented RFC process](rfc/.)
 
-For small changes, RFCs are not required, but for larger changes, RFC's are
-  encouraged. You are welcome to drop into the Slack channel and talk to the community,
-if you want to test the waters, before proposing.
+For small changes, RFCs are not required, but for larger changes, they are
```
Comment (AlekhyaSasi) [inline README.md:273]:
```suggestion
For small changes, RFCs are not required. They are encouraged for larger changes.
```

File: rsts/deployment/cluster_config/general.rst [hunk 2]
```diff
@@ -71,9 +71,9 @@ You can view all custom task-resource-attributes by visiting ``protocol://<host/
 
 Cluster Resources
 =================
-These are free-form key-value pairs that are used when filling in the templates that the admin feeds into the process that syncs kubernetes resources, also known as the cluster manager.
-The keys represent templatized variables in `cluster resource template YAML <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L737,L760>`__
-and the values are what you want to see filled in.
+These are free-form key-value pairs that are used when filling in the templates that the admin feeds into its cluster manager.
```
Comment (AlekhyaSasi) [inline rsts/deployment/cluster_config/general.rst:74]:
```suggestion
These are free-form key-value pairs used when filling in the templates that the admin feeds into its cluster manager. 
```

File: rsts/index.rst [hunk 3]
```diff
@@ -65,7 +65,7 @@ What is Flyte?
 
 Flyte is an open-source, container-native, structured programming and distributed processing platform implemented in Golang. It enables highly concurrent, scalable and maintainable workflows for machine learning and data processing.
 
-Created at `Lyft <https://www.lyft.com/>`__ in collaboration with Spotify, Freenome and many others, Flyte provides first class support for Python, JAVA, and Scala, and is built directly on Kubernetes for all the benefits containerization provides: portability, scalability, and reliability.
+Created at `Lyft <https://www.lyft.com/>`__ in collaboration with Spotify, Freenome and many others, Flyte provides first class support for Python, JAVA, and Scala. It is built directly on Kubernetes for all the benefits of containerization like portability, scalability, and reliability.
```
Comment (AlekhyaSasi) [inline rsts/index.rst:68]:
```suggestion
Created at `Lyft <https://www.lyft.com/>`__ in collaboration with Spotify, Freenome, and many others, Flyte provides first-class support for Python, JAVA, and Scala. It is built directly on Kubernetes for all the benefits of containerization like portability, scalability, and reliability.
```

File: README.md [hunk 4]
```diff
@@ -256,8 +256,8 @@ The table below has different release versions as the columns and the result of
 </details>
 
 <html>
-<h2 id="rfc">
-   🛣️  RFC's (Request for Comments) & Proposals
+<h2 id="rfc"> 
+   🛣️  RFCs (Request for Commments) & Proposals
```
Comment (samhita-alla) [inline README.md:260]:
```suggestion
   🛣️  RFCs (Request for Comments) & Proposals
```

File: rsts/deployment/cluster_config/general.rst [hunk 5]
```diff
@@ -71,9 +71,9 @@ You can view all custom task-resource-attributes by visiting ``protocol://<host/
 
 Cluster Resources
 =================
-These are free-form key-value pairs that are used when filling in the templates that the admin feeds into the process that syncs kubernetes resources, also known as the cluster manager.
-The keys represent templatized variables in `cluster resource template YAML <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L737,L760>`__
-and the values are what you want to see filled in.
+These are free-form key-value pairs used when filling in the templates that the admin feeds into its cluster manager.
```
Comment (samhita-alla) [inline rsts/deployment/cluster_config/general.rst:74]: Can we also have the aka text?

File: rsts/deployment/cluster_config/general.rst [hunk 6]
```diff
@@ -71,9 +71,9 @@ You can view all custom task-resource-attributes by visiting ``protocol://<host/
 
 Cluster Resources
 =================
-These are free-form key-value pairs that are used when filling in the templates that the admin feeds into the process that syncs kubernetes resources, also known as the cluster manager.
-The keys represent templatized variables in `cluster resource template YAML <https://github.com/flyteorg/flyte/blob/1e3d515550cb338c2edb3919d79c6fa1f0da5a19/charts/flyte-core/values.yaml#L737,L760>`__
-and the values are what you want to see filled in.
+These are free-form key-value pairs used when filling in the templates that the admin feeds into its cluster manager. 
+The keys represent templatized variables in `cluster resource template YAML <:admin/sampleresourcetemplates>`__ and the values are what you want to see filled in.
```
Comment (samhita-alla) [inline rsts/deployment/cluster_config/general.rst:75]: The modified link doesn't work.

**Review by samhita-alla — APPROVED** [review]

**Pull request merged**